demo-web/shaders/shader.vert.gles

```diff
@@ -15,5 +15,7 @@ void main()
 {
     interface_var_0 = in_uv;
     gl_Position = vec4(pos, 0.0, 1.0);
```

Comment:
What's going on here? Is this how people would have to write GL shaders? Or is this the generated shader code?

Reply:
the shader processor inserts this fix automatically